templates/kubernetes.html

```diff
@@ -0,0 +1,570 @@
+{% extends "_base/base.html" %}
+
+{% block title %}人工知能/機械学習 | Ubuntu{% endblock %}
```

Comment:
Needs updating - the page title is probably too long so maybe:
```suggestion
{% block title %}Kubernetes | Ubuntu{% endblock %}
```